contracts/Dispatcher.sol

```diff
@@ -83,8 +83,8 @@ contract Dispatcher is IbcDispatcher, Ownable {
         address indexed writerPortAddress,
         bytes32 indexed writerChannelId,
         uint64 sequence,
-        uint64 pktTimeoutHeight,
-        uint64 pktTimeoutTimestamp
+        Height timeoutHeight,
```

Comment:
We can use `Height` type, or we can use `uint64`. 

There is no much difference unless an EVM chain support revision as well.  (I doubt it).

It's fine to keep `Height` here.

Reply:
True.
OTOH, when `WriteTimeoutPacket` is emitted, the running chain is B.  The timeout params here are from the original packet on chain A. So we include them as-is for completeness. Whether the `revision_number` makes sense or should be 0 are determined by IBC clients on Polymer.